src/software/ai/hl/stp/play/play_factory_test.cpp

```diff
@@ -13,7 +13,8 @@ class PlayFactoryTest : public testing::Test
 
 TEST_F(PlayFactoryTest, test_shoot_or_pass_play)
 {
-    TbotsProto::Play play_proto = TbotsProto::Play::ShootOrPassPlay;
-    std::unique_ptr<Play> play  = createPlay(play_proto, ai_config);
+    TbotsProto::Play play_proto = TbotsProto::Play();
+    play_proto.set_name(TbotsProto::Play::PlayName::Play_PlayName_ShootOrPassPlay);
```

Comment:
```suggestion
    play_proto.set_name(TbotsProto::Play::ShootOrPassPlay);
```
Example here https://developers.google.com/protocol-buffers/docs/cpptutorial#enums-and-nested-classes should work with proto2

Reply:
oh cool, I didn't know about that - updating